File: 0088-merge-sorted-array/0088-merge-sorted-array.cpp
class Solution {
public:
    void merge(vector<int>& nums1, int m, vector<int>& nums2, int n) {
        vector<int> result;
        for(int i=0;i<m+n;i++){
            result.push_back(i < m ? nums1[i] : nums2[i-m]);
        }
        nums1 = result;
        sort(nums1.begin(), nums1.end());
        return;
    }
};

File: 0088-merge-sorted-array/0088-merge-sorted-array.py
class Solution:
    def merge(self, nums1: List[int], m: int, nums2: List[int], n: int) -> None:
        """
        Do not return anything, modify nums1 in-place instead.
        """
        for i in range(m+n):
            if nums1[i] == 0 :
                n -= 1
                if n < 0 : break
                nums1[i] = nums2[n]
        nums1.sort()
        return nums1
        